src/applications/personalization/profile360/components/ProfileView.jsx

```diff
@@ -84,6 +85,7 @@ class ProfileView extends React.Component {
                 militaryInformation={militaryInformation}
               />
               <ContactInformation />
+              <PaymentInformation />
```

Comment:
`PaymentInformation` requires the service `evss_claims` for it to be relevant to the user, which is a unique requirement from the other components rendered here. So, I decided to make `PaymentInformation` a container instead, so that I didn't have to pass along more props down the line.

Reply:
cool. this is the first time I've really looked at the `ProfileView` code, but I've noticed that on the Dashboard we are kind of all over the place with regard to determining which components we render; sometimes the logic is abstracted away in the component itself, sometimes the Dashboard itself knows if it should show or hide the subcomponent. It would be nice to be consistent, but I'd need to think a lot more about the ramifications of that.

Reply:
Yeah, the Personalization apps went through a lot of transformation in a short time period. The Profile completely completely changed as we switched from PCIU to the transaction-based Vet360. They have been cleaned up significantly since then but obviously they aren't perfect.

Reply:
In some follow-up work for this, I'll try to clean the Profile a bit, but I don't have the bandwidth right now.

Reply:
I wouldn't worry about that. It's not a priority, IMO, more of a "fix stuff as you go along" kind of thing.